docker-compose.yml

```diff
@@ -1,37 +1,26 @@
---- 
-networks: 
-  firefly_iii_net: 
-    driver: bridge
-services: 
-  firefly_iii_app: 
+version: '3.3'
+
+services:
+  firefly_iii:
     image: jc5x/firefly-iii:latest
+    volumes:
+      - firefly_iii_export:/var/www/firefly-iii/storage/export
+      - firefly_iii_upload:/var/www/firefly-iii/storage/upload
+    env_file: .env
+    ports:
+      - 80:80
     depends_on:
       - firefly_iii_db
-    networks: 
-      - firefly_iii_net
-    ports: 
-      - "80:80"
-    env_file: .env
-    volumes: 
-      - 
-        source: firefly_iii_export
-        target: /var/www/firefly-iii/storage/export
-        type: volume
-      - 
-        source: firefly_iii_upload
-        target: /var/www/firefly-iii/storage/upload
-        type: volume
-  firefly_iii_db: 
-    image: "postgres:10"
+  
+  firefly_iii_db:
+    image: postgres:10-alpine
```

Comment:
This should be bumped to `12`.

Reply:
Is this really needed rn? If not someone could just include it in the next PR since I don't see a reason to create a PR for a version increase unless it would break something.

Reply:
12 is not backwards compatible so it would break everybody’s database.

Reply:
>Is this really needed rn?

no

>12 is not backwards compatible so it would break everybody’s database.

I don't see anyone using this file straight from this repo and breaking their setup trough this. You are expected to download the file and customize it according to your setup but all new users are now on an old postgres version which there is no need to be on. Firefly works great with 12 and I am running it with all my containers since weeks without any issues.

Reply:
The last time I did this that's exactly what happened. People updated without looking and destroyed their databases. So please forgive me for being careful. I'll plan something with an announcement and everything. Just changing this out of the blue will really mess things up for a lot of people, even though it shouldn't.

Reply:
I think it would be a good idea to plan this for the next major version increase or something similar so it won't break and most people can prepare.

Reply:
> destroyed their databases

well this is not true. Postgres just refuses to do anything cause the version mismatches.